# Generated by Django 3.0.6 on 2020-05-31 17:03

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0007_auto_20200531_0043'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='birth_date',
            field=models.DateField(blank=True, null=True, verbose_name='birth day'),
        ),
        migrations.AlterField(
            model_name='user',
            name='country',
            field=models.TextField(blank=True, null=True, verbose_name='Country'),
        ),
        migrations.AlterField(
            model_name='user',
            name='email',
            field=models.EmailField(error_messages={'unique': 'Ops!, This email already exists.'}, max_length=254, unique=True, validators=[django.core.validators.RegexValidator(message='Make sure your mail looks something like this: user@domain.com.', regex='(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$)')], verbose_name='email address'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'Ops!, username already exists.'}, max_length=250, unique=True, verbose_name='username'),
        ),
    ]
